src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorExamplesTest.java

```diff
@@ -88,11 +88,11 @@ public void testExample4() throws Exception {
     @Test
     public void testExample5() throws Exception {
         final String[] expected = {
-            "16:1: " + getCheckMessage(MSG_SHOULD_BE_SEPARATED, "package"),
```

Comment:
Some XML configs become longer than 100 characters when `<`,`>` are substituted with `&lt;`,`&gt;`. I had to split those XML properties in multiple lines and that's why test lines change.

Reply:
Moved to https://github.com/checkstyle/checkstyle/pull/13394

Reply:
Let's not fight with 100 limit too much, we can exclude it from validation or give it some over limit like 120. So simply suppress until end of project to see dull picture at the end.